doc/howto.md

```diff
@@ -110,3 +110,42 @@ Pass existing engines to `%sql`
 %%sql
 SELECT MYSUM(1, 2)
 ```
+
+## Connect to a SQLite database with spaces
+
+Currently, due to a limitation in the argument parser, it's not possible to directly connect to SQLite databases whose path contains spaces; however, you can do it by creating the engine first.
```

Comment:
We're keeping the issue open on this? It will probably be a quick fix.

Reply:
It isn't a simple fix. My guess is that it's related to this: https://github.com/ploomber/jupysql/issues/33 (the parsing logic hasn't been well tested)

Reply:
Ok we'll keep it open until solved.